### DISCUSSION TITLE
useApolloClient not working inside NodeViewWrapper

### DISCUSSION THREAD

**Discussion by amit-verma-qatalog** [ueberdosis/tiptap#2078] [General Questions]
I am trying to plug a react component inside the tiptap editor and it involves making graphql calls to apollo client. 
now since the NodeViewWrapper instances are wrapped in portals, I want to know how can I make useApolloClient work? Is there a way? 

Interestingly the `useHistory` hook works. The `<Router>` is higher up the order. Why is apollo failing?

**Comment by philippkuehn** [comment]
I’m not sure. All node views are rendered as portals inside of `<EditorContent>` so this should work. Can you create a reproducable codesandbox for that?

**Reply by amit-verma-qatalog** [reply]
its hard to create a sandbox to properly reproduce this. but I am building `mention` tags and i wanted to fetch data via graphql within those mention tags itself.
but i could do a `new ApolloClient` instance and start making graphql calls. leaving this here in case someone comes here searching for the same issue.

**Reply by amit-verma-qatalog** [reply]
@philippkuehn thanks for replying, though. i appreciate it!

**Reply by yjun1989** [reply]
thank you for leaving this, i have the same issue, build mention tags and fetch data via graphql
